packages/preview-middleware-client/src/adp/init.ts

```diff
@@ -60,7 +64,37 @@ export default async function (rta: RuntimeAuthoring) {
     const ui5VersionValidationMsg = getUI5VersionValidationMessage(version);
 
     if (ui5VersionValidationMsg) {
-        sendAction(showMessage(ui5VersionValidationMsg));
+        sendAction(showMessage({ message: ui5VersionValidationMsg, shouldHideIframe: true }));
+
+        return;
+    }
+    if (syncViewsIds.length > 0) {
+        sendAction(
+            showMessage({
+                message:
+                    'Have in mind that synchronous views are detected for this application and controller extensions are not supported for such views. Controller extension functionality on these views will be disabled.',
+                shouldHideIframe: false
+            })
+        );
     }
+
     log.debug('ADP init executed.');
 }
+
+/**
+ *
+ * Get Ids for all sync views
+ *
+ * @returns array
+ */
+function getAllSyncViewsIds(): string[] {
```

Comment:
Extra space here in the comment. Returns could have a better explanation :)

Reply:
Done